test/evergreen.yml

```diff
@@ -468,7 +469,7 @@ functions:
           pip3 install google-cloud-storage
 
           # Run Python testing for all tiered tests.
-          python3 ../test/suite/run.py -j $(nproc) ${tiered_storage_test_name}
+          python3 ../test/suite/run.py --ignore-stdout -j $(nproc) ${tiered_storage_test_name}
```

Comment:
Why a bunch of `--ignore-stdouts`?

Reply:
Slower WiredTiger execution was producing `session sweep took more than 60 minutes` or `generation drain waited XXX minutes` warning logs. WiredTiger still behaves correctly, but the test detects content in stdout/stderr and fails the test on that basis. I initially added this to distinguish between tests becoming verbose and tests that were actually failing.

Reply:
Cleaned up in [43be9c4](https://github.com/wiredtiger/wiredtiger/pull/10812/commits/43be9c45d59fcc73e1e0c7c980b95c07149f545a)